src/rollup/DeployHelper.sol

```diff
@@ -0,0 +1,125 @@
+
+    uint256 internal constant GASLIMIT = 100_000;
+    uint256 internal constant MAXFEEPERGAS = 1_000_000_000;
```

Comment:
I have this comment https://github.com/OffchainLabs/nitro-contracts/pull/51#discussion_r1307443800 on another pr about maybe supplying this as params

Reply:
implemented here https://github.com/OffchainLabs/nitro-contracts/pull/59/commits/03c98c03c954366dfeab61a21068e6764ad3588a 

maxFeePerGas is provided as param to DeployHelper,  and also to RollupCreator when new rollup is to be created